Libs/MRML/Core/vtkMRMLTransformDisplayNode.h

```diff
@@ -49,6 +49,7 @@ class VTK_MRML_EXPORT vtkMRMLTransformDisplayNode : public vtkMRMLDisplayNode
     VIS_MODE_GLYPH,
     VIS_MODE_GRID,
     VIS_MODE_CONTOUR,
+    VIS_MODE_INTERACTIVE,
```

Comment:
You can add EditorWidgetVisibility and EditorWidgetSliceVisibility flags and handle those completely independently from the Visibility and SliceIntersectionVisibility flags.

Also, please add a separate displayable manager for the widget (similar to vtkMRMLThreeDReformatDisplayableManager) that only checks EditorWidgetVisibility and EditorWidgetSliceVisibility flags and the transform type. A separate displayable manager is appropriate, because the widget only supports linear transforms (while the transform displayable manager supports all transform types) and we'll have to add separate widgets for TPS, Bspline, etc. It also results in having two simpler, independent classes instead of forcing two completely unrelated features into one class.